mgmt/tribe/worker/worker.go

```diff
@@ -545,14 +545,49 @@ func (w worker) isPluginLoaded(n, t string, v int) bool {
 }
 
 func getSchedule(s *core.Schedule) schedule.Schedule {
```

Comment:
Still "nil" is returned if something was going wrong, so tribe-logs will point to invalid nil schedule as it works currently.